### ISSUE TITLE
Doc (?): Comparison to Slate

### ISSUE THREAD

**Issue by gibiansky** [sdegutis/mjolnir#113]
Hey there! This looks like a really, really cool project, and I really like the style and general idea. However, I've been a Slate user for a while, which serves a very similar purpose. I (and I think many others in a similar boat) would be happy to try out Hydra, but need _some_ reason to try it over our default existing configs.

What does Hydra do better than Slate? What can I do in Hydra that I can't in Slate? Lua scripting is a good example here. Can Hydra do things with spaces? Mouse? Etc? 

Basically: Great project, but I would really love a bit of a motivation as to why I should switch to Hydra from something else :) (I'm going to try it soon anyways, because it looks awesome, but just figured I'd note some of my thinking.)

**Comment by sdegutis** [comment]
@gibiansky Thanks for the suggestion. I'll add a little blurb on the readme to compare them.

**Comment by sdegutis** [comment]
For the record, this is probably the hardest task I have yet done for Hydra. Writing comparisons is hard!

**Comment by sdegutis** [comment]
@gibiansky See issue #94 for discussion about modal modes. You can't change the icon color right now but you could display a temporary borderless/shadowless textgrid in the corner instead, to show you your current mode.
